core/arch/arm/plat-stm32mp2/stm32_sysconf.h

```diff
@@ -0,0 +1,105 @@
+/* SPDX-License-Identifier: BSD-2-Clause */
+/*
+ * Copyright (c) 2024, STMicroelectronics
+ */
+
+#ifndef __STM32_SYSCONF_H__
+#define __STM32_SYSCONF_H__
+
+#include <assert.h>
```

Comment:
```patch
-#include <assert.h>
-#include <types_ext.h>
+#include <util.h>
```

_stdbool.h_ should be added in the next commit of this series. I think it's ok to keep it here but it would be cleaner to introduce it when needed.